MediaBrowser.Common/Net/IPNetAddress.cs

```diff
@@ -0,0 +1,277 @@
+#nullable enable
+using System;
+using System.Net;
+using System.Net.Sockets;
+
+namespace MediaBrowser.Common.Net
+{
+    /// <summary>
+    /// An object that holds and IP address and subnet mask.
+    /// </summary>
+    public class IPNetAddress : IPObject
+    {
+        /// <summary>
+        /// Represents an IPNetAddress that has no value.
+        /// </summary>
+        public static readonly IPNetAddress None = new IPNetAddress(IPAddress.None);
+
+        /// <summary>
+        /// IPv4 multicast address.
+        /// </summary>
+        public static readonly IPAddress MulticastIPv4 = IPAddress.Parse("239.255.255.250");
```

Comment:
Is this always the case?

Reply:
Yes - but i've renamed to clarify futher.